fgd/bases/Door.fgd

```diff
@@ -21,6 +21,13 @@
 		"if the player tries to block them with objects."
 	ignoredebris(boolean) : "Ignore Debris" : 0 : "If set this will change the door's collision group to one that ignore collisions with debris objects " +
 		"(note that this is not compatible with the non-solid-to-player spawnflag)."
+	
+	solidbsp[engine](boolean) : "Solid Type" : 1
+	solidbsp(choices) : "Solid Type" : 1 =
```

Comment:
This should copy the description from the original line so mappers know what this changes. (The note about BSP being more rudimentary but more correct collisions).

We also need to default to 1 for these, as reference: https://github.com/momentum-mod/game/commit/79617c9be2c09af0d241ebcdd0eeb8939ee75f23

It shouldn't break anything but future maps, and does not need to be game dependent.

Reply:
> This should copy the description from the original line so mappers know what this changes. (The note about BSP being more rudimentary but more correct collisions).

To be clear, `"Set this if this brush is in heirarchy with a moving object of some kind, and the player can stand on this brush."` ?
I see that it was removed in that commit but no other description was added.

Reply:
`"How to calculate collisions. BSP is more rudimentarty but allows more stable collision with players."`